frontend/workflows/projectSelector/src/dash.tsx

```diff
@@ -40,7 +46,232 @@ const Dash = ({ children }) => {
       <DashDispatchContext.Provider value={dispatch}>
         <DashStateContext.Provider value={state}>
           <ProjectSelector />
-          <CardContainer>{children}</CardContainer>
+          <CardContainer>
```

Comment:
as usual, nothing in this file will be merged. just for PR reviewers if they'd like to test as well :)